src/bosquet/template/tag.clj

```diff
@@ -13,9 +13,7 @@
 (defn add-tags []
   (parser/add-tag! :llm-generate
     (fn [args context]
-      (openai/complete (:selmer/preceding-text context) 
-                       (merge 
-                        (args->map args)
-                        (get-in context [:opts (-> context :the-key) :llm-generate])
```

Comment:
@behrica, maybe you could comment on this change? at least for me, the model options would not be passed correctly here

Reply:
this allows to override externally the args of `llm-generate`.
It needs to be seen together with this line:
https://github.com/zmedelis/bosquet/blob/d150d4f783eefd8e554fda59a2c3ccdf2db0ee40/src/bosquet/generator.clj#L45

So i use a "magic keywored"   `:the-key`  to pass the options down and merge them.

Example usage of the arg overriding is here:
https://github.com/zmedelis/bosquet/blob/d150d4f783eefd8e554fda59a2c3ccdf2db0ee40/notebook/use_guide.clj#L163